### ISSUE TITLE
CRB apps: Issues with building

### ISSUE THREAD

**Issue by aancel** [feelpp/feelpp#727]
I am encountering issues building CRB apps. Here is the compilation log with crb_heat1dapp:
```
[ 20%] Built target feelpp_metis
[ 40%] Built target feelpp_ginac
[ 60%] Built target feelpp_nlopt
[100%] Built target feelpp
[100%] Building CXX object applications/crb/heat1d/CMakeFiles/crb_heat1dapp.dir/heat1dapp.cpp.o
In file included from /home/u2/ancel/git/feelpp/applications/crb/heat1d/heat1dapp.cpp:2:
In file included from /home/u2/ancel/git/feelpp/applications/crb/heat1d/heat1d.hpp:38:
In file included from /home/u2/ancel/git/feelpp/feel/feelcrb/modelcrbbase.hpp:33:
In file included from /home/u2/ancel/git/feelpp/feel/feelcrb/eim.hpp:58:
/home/u2/ancel/git/feelpp/feel/feelcrb/crbmodel.hpp:827:38: error: no member named 'assemble' in 'Feel::Heat1D'
    void assemble(){ return M_model->assemble(); }
                            ~~~~~~~  ^
/home/u2/ancel/git/feelpp/feel/feelcrb/opusapp.hpp:451:28: note: in instantiation of member function 'Feel::CRBModel<Feel::Heat1D>::assemble' requested here
                    model->assemble(); //Affine decomposition has changed since eim has changed
                           ^
/home/u2/ancel/git/feelpp/feel/feelcrb/opusapp.hpp:494:23: note: in instantiation of member function 'Feel::OpusApp<Feel::Heat1D, CRB, CRBModel>::SER' requested here
                this->SER(); // Simultaneous EIM - RB
                      ^
/home/u2/ancel/git/feelpp/applications/crb/heat1d/heat1dapp.cpp:22:9: note: in instantiation of member function 'Feel::OpusApp<Feel::Heat1D, CRB, CRBModel>::run' requested here
    app.run();
        ^
1 error generated.
make[3]: *** [applications/crb/heat1d/CMakeFiles/crb_heat1dapp.dir/heat1dapp.cpp.o] Error 1
make[2]: *** [applications/crb/heat1d/CMakeFiles/crb_heat1dapp.dir/all] Error 2
make[1]: *** [applications/crb/heat1d/CMakeFiles/crb_heat1dapp.dir/rule] Error 2
make: *** [applications/crb/heat1d/CMakeFiles/crb_heat1dapp.dir/rule] Error 2
```

Everything was working fine on the branch I am currently working on (feature/crb-hdf5-db), until I made a merge today.
I think that it might come from this commit https://github.com/feelpp/feelpp/commit/c74dca18f2c109db76e4f11c0e0f0e061022815e, could you check if the code works on your side @cdaversin ?

Thanks in advance !

**Comment by cdaversin** [comment]
@aancel Thanks for your feedback. Indeed I forgot that some models don't have `assemble()` function as the one you mention. The problem should be resolved with my last commit 50599878d3b211851a2ecbd133f09666da52aad2.
I just tested compilation (with gcc-5) and execution of crb_heat1dapp application, it seems to be ok now. Let me know if it fixes your issue.

**Comment by aancel** [comment]
@cdaversin Thanks for the fix ! It is ok now. I'll close this issue.
